### PR TITLE
Added docs support for Dropdown component.

### PR DESCRIPTION
**Pull request by MaximFedarau**
This component  already exists, but it does not have a storybook setup.
I writed docs and set up playground.

### CODE REVIEW

**Comment by MaximFedarau** [comment]
@tripurari001 @rahuldkjain @mittalchirag

**Comment by MaximFedarau** [comment]
@tripurari001 @rahuldkjain @mittalchirag

File: src/components/Dropdown/Dropdown.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    title="Components/Dropdown"
+    component={Dropdown}
+    argTypes={{ onClick: { action: 'clicked' } }}
+/>
+
+export const StorybookDropdown = (args) => <Dropdown {...args} />;
+
+# Dropdown
+
+Drowdown allows to show interactive content.
```
Comment (shanpriyan) [inline src/components/Dropdown/Dropdown.stories.mdx]: typo, Drowdown -> Dropdown

Comment (MaximFedarau) [reply src/components/Dropdown/Dropdown.stories.mdx]: Hello. Thank you very much for the review. I fixed that typo.

**Comment by MaximFedarau** [comment]
@tripurari001 @rahuldkjain @mittalchirag @shanpriyan

**Comment by MaximFedarau** [comment]
@tripurari001 @rahuldkjain @mittalchirag @shanpriyan

**Pull request closed without merging**